src/encoded/tests/data/master-inserts/higlass_view_config.json

```diff
@@ -689,9 +663,9 @@
                         "y": 6,
                         "w": 6,
                         "x": 0,
-                        "i": "93a36311-2eb5-4c74-9663-6d2be03ae348",
                         "h": 6,
                         "static": false
+                        
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Trailing spaces not allowed. (no-trailing-spaces)](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=5006697)